test cases/windows/5 resources/meson.build

```diff
@@ -1,9 +1,66 @@
 project('winmain', 'c')
 
-win = import('windows')
-res = win.compile_resources('myres.rc',
-  include_directories : include_directories('inc')
-)
+# MinGW windres has a bug due to which it doesn't parse args with space properly:
+# https://github.com/mesonbuild/meson/pull/1346
+# https://sourceware.org/bugzilla/show_bug.cgi?id=4933
+if meson.get_compiler('c').get_id() == 'gcc'
+  # Construct build_to_src and skip this test if it has spaces
+  # because then the -I flag to windres will also have spaces
+  # and we know the test will fail
```

Comment:
Isn't this always the case? The build dir used always has a space and the path to the source tree is `test cases/windows/5 resources/`.